safe-room.lic

```diff
@@ -32,18 +32,7 @@ class SafeRoom
       pc_empaths << { 'name' => settings.safe_room_empath, 'id' => settings.safe_room_id } unless pc_empaths.any? { |empath| empath['name'] == settings.safe_room_empath && empath['id'] == settings.safe_room_id }
     end
 
-    if !pc_empaths.empty?
-      # Sort pc empaths list by nearest first
-      sort_destinations(pc_empaths.map { |empath| empath['id'] })
-        .map { |room| pc_empaths.select { |empath| empath['id'] == room } }
-        .flatten
-        .each do |empath|
-        next unless DRRoom.pcs.include?(empath['name'].capitalize!) || in_game?(empath['name'])
-        next unless use_pc_empath?(empath['id'], empath['name'])
-        ensure_copper_on_hand(settings.safe_room_tip_threshold || 0, settings)
-        tip(settings.safe_room_tip_threshold, settings.safe_room_tip_amount, empath['name'])
-        break
-      end
+    if use_pc_empaths?(pc_empaths, settings)
```

Comment:
💡 What do you think about adding a comment to note this is deliberately empty?

Reply:
That sounds good to me.